clang/lib/Sema/SemaChecking.cpp

```diff
@@ -17626,6 +17626,25 @@ class SequenceChecker : public ConstEvaluatedExprVisitor<SequenceChecker> {
     for (unsigned I = 0; I < Elts.size(); ++I)
       Tree.merge(Elts[I]);
   }
+
+  void VisitCXXParenListInitExpr(const CXXParenListInitExpr *PLIE) {
+    // C++20 parenthesized list initializations are sequenced. See C++20
+    // [decl.init]p17.5 and [decl.init]p17.6.2.2
```

Comment:
I think you mean `dcl.init.general`, not `decl.init` here, and the wording is in p16 not p17 in C++20 and the current working paper.
```suggestion
    // [dcl.init.general]p16.5 and [dcl.init.general]p16.6.2.2.
```